test/mitmproxy/test_examples.py

```diff
@@ -209,3 +209,21 @@ def test_binary(self):
                 har = json.load(inp)
 
         assert len(har["log"]["entries"]) == 1
+
+    def test_empty_bytes(self):
```

Comment:
We should rather have new tests in https://github.com/mitmproxy/mitmproxy/blob/master/test/mitmproxy/net/http/test_encoding.py. I just looked at what's there already and that doesn't make much sense to me. gzip/deflate/br should only be tested on bytes (they are byte -> byte transformations), and apparently there are no tests for byte->str encodings like utf8. If you feel comfortable doing that:

- remove str testing for gzip/deflate/br in test_encoders so that it only tests those encodings on bytes.
- copy test_encoders for utf8/latin1 and test byte->str encoding there.
- add docstrings explaining that.
- remove this test here, it tests no special case if the encodings module works as intended.